Assets/HoloToolkit/SpectatorView/Scripts/SpatialSync/WorldSync.cs

```diff
@@ -0,0 +1,298 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.using UnityEngine;
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.Networking;
+
+namespace HoloToolkit.Unity.SpectatorView
```

Comment:
Seems like this could be in the Sharing With UNET feature folder instead.

Reply:
We'll refactor for vNext

Reply:
I think some more thought into how this should be implemented can/should happen now before it's part of the repository.